message-markdown.md

```diff
@@ -0,0 +1,44 @@
+# Markdown in messages
+
+Deltachat is going to support the following subset of markdown:
+
+### `**bold**`
+
+### `__underline__`
```

Comment:
i would suggest to format words surrounded by one or two underscores as _italics_. i would not support underline at all as this can easily be confused with links.